applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_compressible_potential_element.cpp

```diff
@@ -0,0 +1,333 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Inigo Lopez
+//
+//
+
+// System includes
+#include <set>
+
+// External includes
+#include "containers/model.h"
+
+// Project includes
+#include "testing/testing.h"
+#include "includes/model_part.h"
```

Comment:
I think that the `model.h`already includes this.

Reply:
it does